xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml

```diff
@@ -53,8 +53,7 @@
     #set ($useWysiwygEditor = $xwiki.getUserPreference('editor') == 'Wysiwyg')
   #end
   {{html clean="false"}}
-  ## The "content" id is expected by some JavaScript and CSS code.
-  #set ($id = 'content')
+  #set ($id = "${prefix}${name}1")
```

Comment:
Is this really the correct ID? Shouldn't this number be based on some variable?

Reply:
Okay, it's just stored in the `$name` variable we get from the context in which those templates are used. 
I could find this thanks to `displayer_date.vm` which is not defined the same way as those two templates, but has a similar role.
Note that in both of those templates, the value `$name` is overridden, we want to compute with the info from the context so we have to compute the ID at the start of the template.

I just used `#set ($id = $escapetool.xml("${prefix}${name}"))` on both and we're good :)
This works both on classEdition and entryEdition.